roles/kubevirt/tasks/provision.yml

```diff
@@ -25,41 +25,21 @@
   command: "oc adm policy add-scc-to-user hostmount-anyuid -z kubevirt-infra -n {{ namespace }}"
   when: cluster=="openshift"
 
-- name: Check for kubevirt.yaml template in {{ kubevirt_template_dir }}
+- name: Check for kubevirt.yaml.j2 template in {{ kubevirt_template_dir }}
   stat:
-    path: "{{ kubevirt_template_dir }}/kubevirt.yaml"
+    path: "{{ kubevirt_template_dir }}/kubevirt.yaml.j2"
   register: byo_template
 
```

Comment:
I think we need to keep these tasks so offline works for upstream images. They will need to change to work with the .j2 extension.
```-- name: Check for offline v{{ version }} templates in {{ offline_template_dir }}```
```-- name: Copy offline templates to /tmp```

Reply:
Adding a couple commits to address these. Thanks!